### ISSUE TITLE
Edits/suggestions from Jaclyn (SRG 2019 meeting)

### ISSUE THREAD

**Issue by andrew-edwards** [cgrandin/hake-assessment#577]
Attached are minor edits/ suggestions for the Exec Summary to clarify wording/ interpretation.  

I think 5) and 7) are the only ones necessary for SRG discussion. 

Questions/ edits for Executive Summary

- [x] 1.	Page 5
“The stock is estimated to have been at relatively high levels since 2013…”

Suggest linking this statement directly to %Bo. (rather than subjective statement)

- [x] 2.	Page 11
Suggest: “Pacific Hake appear to have low average recruitment with occasional large year-classes (Table c and Figure e).”

- [x] 3.	Page 12
Suggest: “From 2000 to 2007 estimated recruitment was at some of the lowest values in the time series, but this was followed by a relatively large an above average 2008 year class.”

- [x] 4.	Page 14
Suggest: “Total landings last exceeded the coast-wide quota in 2002 when utilization was 112%, though in retrospect, the fishing intensity was relatively low in that year due to appearance of the 1999 year class.”

- [x] 5.	Page 15

**The joint history of the medians for relative spawning biomass and relative fishing intensity shows that the median relative fishing intensity has never been above the target of 1.0 when the female spawning biomass is below the reference point of B40% (Figure h).** This highlights the highly dynamic nature of the stock due to high variation in recruitment strength, **such that spawning biomass could fall below B40% even in the absence of fishing**.

I’m not following how the second statement in bold is explained by the first statement in bold.

- [x] 6.	Page 17 (2nd para)
“However, comparable results in terms of catch could be achieved with a more precise age-2+ survey or alternative harvest control rules.”

Was this shown?
If no, suggest rewording to indicate “may be achieved…”

- [x] 7.	Relative fishing intensity:

Relative fishing intensity is described as a ratio from 0:1 but expressed as both a percentage and ratio. 
e.g., “Median relative fishing intensity on the stock is estimated to have been below the target of 1.0 for all years” (p13), Fig f
and
“Between 2007 and 2011, median relative fishing intensity ranged from 80% to 97%” (p15), Table h

Suggestion: add a statement to clarify that ratio and percentage are reported interchangeably for this metric.

**Comment by andrew-edwards** [comment]
7 - be good to make it consistent. Maybe best for general discussion in the meeting to decide what people prefer?

**Comment by kellijohnson** [comment]
Fixed 1 with d82aaf010ce522a2b670d7835769fb6e03982299

**Comment by kellijohnson** [comment]
Jaclyn said to remove "such ..." during her formal request for 5. Also did 2-6. Someone else can handle 7 :).
Done with a9d93338219ce4a00bbb0906f2edfe6f0f77585c

**Comment by andrew-edwards** [comment]
I can build again (just not without the new catch levels as I can't run SS3 any more). But I can do text edits and check them... back in business. I'll do 7.
